src/faction_camp.cpp

```diff
@@ -208,7 +208,7 @@ std::map<std::string, bcp_miss_data> basecamp_missions_info = {{
         },
         {
             "_faction_exp_chop_shop_", {
-                "Chop Shop", _( "Chop Shop" ), _( "Working at the chop shop...\n" ),
+                " Chop Shop", _( "Chop Shop" ), _( "Working at the chop shop...\n" ),
```

Comment:
Isn't this the same change as #29488?

Reply:
oops yeah. was using this as my local playing copy. and wanted that fix in for myself ha.